### PR TITLE
delete sticker page and add canvas draw

### PR DESCRIPTION
**Pull request by isabellpaulmann**

### CODE REVIEW

File: src/pages/DetailPageDarker.js [hunk 1]
```diff
@@ -45,83 +80,76 @@ export default function DetailPageDarker({wallpapersDark}) {
...
+        ></Canvas>
+        <ButtonBar>
+          <button onClick={startDrawing}>
+            <img src={draw} alt="start drawing" />
+          </button>
+          {visible2 && (
+            <StyledInput
+              type="number"
+              value={lineWidth}
+              onChange={e => setLineWidth(Number.parseFloat(e.target.value))}
```
Comment (pixelass) [inline src/pages/DetailPageDarker.js:102]: You need to set the local state and use the canvasRef method `set` to tell the canvas component to update the lineWidth.

File: src/pages/DetailPageDarker.js [hunk 2]
```diff
@@ -45,83 +80,76 @@ export default function DetailPageDarker({wallpapersDark}) {
...
+                }
+              }}
+            >
+              <img src={deleteAll} alt="delete all drawings" />
+            </button>
+          )}
+        </ButtonBar>
+        {visible && (
+          <StyledColorPicker className="small">
+            <HexColorPicker color={color} onChange={setColor} />
```
Comment (pixelass) [inline src/pages/DetailPageDarker.js:125]: You need to set the local state and use the canvasRef method `set` to tell the canvas component to update the color.

File: src/pages/DetailPageDarker.js [hunk 3]
```diff
@@ -45,83 +80,76 @@ export default function DetailPageDarker({wallpapersDark}) {
...
+        <Canvas
+          id="canvas"
+          color={color}
+          lineWidth={lineWidth}
+          ref={canvasRef}
+          disabled={isDisabled}
+          image={thisWallpaper.image}
           style={{
             filter: `brightness(${imageStyle.brightness}) contrast(${imageStyle.contrast}) saturate(${imageStyle.saturate})`,
           }}
```
Comment (pixelass) [inline src/pages/DetailPageDarker.js:92]: this has no effect and should be done via the setFilter Method.

**Review by stefankorder — APPROVED** [review]
Good work!

**Pull request merged**